scripts/install.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/sh
+#
+# This script installs the latest release of Tenzir.
+#
+
+# Treat unset variables as an error when substituting.
+set -u
+
+#
+# Helper utilities
+#
+
+# Only use colors when we have a TTY.
+if [ -t 1 ]
+then
+  escape() { printf "\033[%sm" "$1"; }
+else
+  escape() { :; }
+fi
+make_bold() { escape "1;$1"; }
+green="$(make_bold 32)"
+blue="$(make_bold 34)"
+bold="$(make_bold 39)"
+normal="$(escape 0)"
+
+# Prints an "action", i.e., something the script is doing.
+# Use this to transparently inform the user about the key
+# steps this script takes.
+action() {
+  printf "${blue}==>${bold} %s${normal}\n" "$@"
+}
+
+#
+# Execution
+#
+
+if [ -z "${BASH_VERSION:-}" ]
+then
+  echo "Bash is required to interpret this script."
+  exit 1
+fi
+
+# Print welcome banner.
+echo "${blue}"
+echo "                _____ _____ _   _ ________ ____  "
+echo "               |_   _| ____| \ | |__  /_ _|  _ \ "
+echo "                 | | |  _| |  \| | / / | || |_) |"
+echo "                 | | | |___| |\  |/ /_ | ||  _ < "
+echo "                 |_| |_____|_| \_/____|___|_| \\_\\"
+echo "${normal}"
+echo "                            INSTALLER"
+echo
+
+# Detect OS.
+action "Identifying platform"
+PLATFORM=
+OS=$(uname -s)
+if [ "${OS}" = "Linux" ]
+then
+    if [ -f "/etc/debian_version" ]
+    then
+        PLATFORM=Debian
+    elif [ -f "/etc/NIXOS" ]
+    then
+        PLATFORM=NixOS
+    else
+        PLATFORM=Linux
+    fi
+elif [ "${OS}" = "Darwin" ]
+then
+  PLATFORM=macOS
+fi
+echo "Found ${PLATFORM}."
+
+# Select appropriate package.
+action "Identifying package"
+PACKAGE=
+if [ "${PLATFORM}" = "Debian" ]
+then
+  PACKAGE="tenzir-linux-static.deb"
+elif [ "${PLATFORM}" = "Linux" ]
+then
+  PACKAGE="tenzir-linux-static.tar.gz"
+elif [ "${PLATFORM}" = "NixOS" ]
+then
+  echo "Try Tenzir with our ${bold}flake.nix${normal}:"
+  echo
+  echo "    ${bold}nix run github:tenzir/tenzir/stable${normal}"
+  echo
+  echo "Install Tenzir by adding" \
+    "${bold}github:tenzir/tenzir/stable${normal} to your"
+  echo "flake inputs, or use your preferred method to include third-party"
+  echo "modules on classic NixOS."
+  exit 0
+else
+  echo "We do not offer pre-built packages for ${PLATFORM}." \
+       "Your options:"
+  echo
+  echo "  1. Use Docker"
+  echo "  2. Build from source"
+  echo
+  echo "Visit ${bold}https://docs.tenzir.com${normal} for further instructions."
+  exit 1
+fi
+echo "Using ${PACKAGE}"
+
+# Download package.
+BASE="https://github.com/tenzir/tenzir/releases/latest/download"
+URL="${BASE}/${PACKAGE}"
+action "Downloading ${URL}"
+curl --progress-bar -L "${URL}" -o "${PACKAGE}" || exit 1
```

Comment:
`curl` is not installed by default in Debian, but `wget` is. I suspect that is the case in some other distros as well.
```suggestion

check() {
  command -v "$1" 2>&1 >/dev/null
}

# Download package.
tmpdir="$(dirname "$(mktemp -u --tmpdir)")"
BASE="https://github.com/tenzir/tenzir/releases/latest/download"
URL="${BASE}/${PACKAGE}"
action "Downloading ${URL}"
if check quizf; then
  exit 1
elif check wget; then
  wget -q --show-progress -O "${tmpdir}/${PACKAGE}" "${URL}"
elif check curl; then
  curl --progress-bar -L -o "${tmpdir}/${PACKAGE}" "${URL}"
else
  echo "Neither 'wget' nor 'curl' in \$PATH, aborting..."
  exit 1
fi
```

Reply:
`quizf` doesn't exist on macOS. (Never heard of it before.) I think `curl` and `wget` are enough, no?

Reply:
I've switched to `wget` and `curl`. Feel free to resolve if that suits the bill.

Reply:
That was a copy-n-paste mistake. I just wanted to make sure the code worked as intended before posting the suggestion.